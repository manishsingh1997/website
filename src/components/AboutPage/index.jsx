import React from 'react';

import {Button} from '@ergeon/core-components';
import imgHeart from 'assets/about-page/ergeon_heart.svg';
import imgJenny from 'assets/about-page/jenny@2x.jpg';
import imgOdysseas from 'assets/about-page/odysseas@2x.jpg';
import imgLogoMark from 'assets/about-page/logo_mark.png';

import './index.scss';

class AboutPage extends React.Component {
  render() {
    return (
      <div className="wrapper-1180">
        <div className="spacing before__is-48 after__is-48">
          <h1 className="center spacing after__is-12">
            Our mission
          </h1>
          <p className="subheader h2 center">
            Simplify home improvement <br/>by empowering skilled local contractors
          </p>
        </div>
        <div className="values-container">
          <div className="values-item">
            <img alt="" className="image-24" src={imgHeart} />
            <h5>We invest in people</h5>
            <p className="values-item__subtitle">Making long term commitments to customers and team members.<br /></p>
          </div>
          <div className="values-item">
            <img alt="" className="image-24" src={imgHeart} />
            <h5 className="values-item__title">We are lean</h5>
            <p className="values-item__subtitle">Maximizing value for customers by minimizing waste.</p>
          </div>
          <div className="values-item">
            <img alt="" className="image-24" src={imgHeart} />
            <h5 className="values-item__title">We are kind</h5>
            <p className="values-item__subtitle">Setting new standards for fairness, empathy and respect.</p>
          </div>
        </div>
        <div className="about-section">
          <h3 className="spacing after__is-24">About Ergeon</h3>
          <p>
            We set out in 2018 to disrupt the multi trillion dollar construction space with technology.
            Using proprietary technology coupled with operational excellence we are building the largest,
            most customer-friendly construction company in the world. We own the consumer-contractor relationship
            end-to-end. Our ‘tech stack’ delivers a seamless experience for consumers and contractors.
            As a result, we increase the build capacity of our contractors and deliver projects 3x faster
            at competitive prices. Starting with outdoor home improvement, we install the highest quality
            fence and hardscape. In time, we will scale to larger and more complex building projects.
            <br/><br/>
            Ergeon was founded and is run today by two CS PhD serial entrepreneurs that share a deep passion for
            utilizing technology to improve how people work. Our innovative technologies help consumers,
            contractors and our all-remote staff excel.
            <br/><br/>
            We believe amazing companies start with amazing people. As an all-remote company (read more &nbsp;
            <a href="https://medium.com/ergeon">here</a>) we know those people come from all over the world.
            We have a fast growing and diverse team spread
            across 20+ countries today. We go to great lengths to ensure that our workforce, everywhere in the
            world, has access to career growth, an amazing work culture and great benefits.
            Come join us to transform the construction industry.<br/><br/>
          </p>
          <div className="special-section">
            <div className="profiles">
              <div className="profile">
                <img alt="" className="profile__image" src={imgJenny} width="96" />
                <div className="profile__info">
                  <h5>Jenny He</h5>
                  <div className="label"><i>Founder and CEO</i></div>
                  <a className="linkedin-link" href="https://www.linkedin.com/in/jiayue-jenny-he-5a10762" rel="noopener noreferrer" target="_blank" >Linkedin</a>
                </div>
              </div>
              <div className="profile">
                <img alt="" className="profile__image" src={imgOdysseas} width="96" />
                <div className="profile__info">
                  <h5>Odysseas Tsatalos</h5>
                  <div className="label"><i>Founder and CTO</i></div>
                  <a className="linkedin-link" href="https://www.linkedin.com/in/tsatalos" rel="noopener noreferrer" target="_blank">Linkedin</a>
                </div>
              </div>
            </div>
            <div className="tile">
              <img alt="" src={imgLogoMark} width="67" />
              <p className="tile__description">
                If you crave an opportunity for impact and rapid career growth, join us!
              </p>
              <Button
                asAnchor
                className="min-width__is-120"
                flavor="secondary"
                href="/careers"
                size="large">Join us!
              </Button>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default AboutPage;
