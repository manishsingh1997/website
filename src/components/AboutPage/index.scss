@import '@ergeon/core-components/src/components/variables.scss';
@mixin skew-container ($width, $height: 100%, $position: absolute, $display: flex) {
  content: '';
  position: $position;
  display: $display;
  border-radius: $corner-soft;
  width: $width;
  height: $height;
  top: 0;
  background-image: linear-gradient(180deg, $blue-3, $blue-2);
}

@mixin skew-display ($right: none, $left: none, $skewX: 0, $zIndex: none, $middle: 0) {
  right: $right;
  left: $left;
  z-index: $zIndex;
  transform: skewX($skewX);

  @if $middle != 1 {
    @include tablet {
      transform: skewX(0);
      display: none;
    }
  }
  @if $middle = 1 {
    transform: skewX(-$skewX);
    > * {
      transform: skewX($skewX);
    }

    @include tablet {
      width: 100%;
      transform: none;
      > * {
        transform: none;
      }
    }
  }
}

.short_line {
  $short_heart: 4px;

  display: block;
  width: $h-60;
  height: $short_heart;
  margin: $v-12 auto;
  background-color: $gray-4;
}
.values-container {
  display: flex;
  max-width: 940px;
  margin: auto;
  justify-content: space-between;
  margin-top: $v-20 * 2;

  .values-item {
    $item-width: 270px;
    $item-skew-distance: -30px;
    $item-max-width: $v-30 * 10;
    $item-height: $v-20 * 10;
    $skew-width: $h-60 + $h-20;
    $corner-custom: 10px;
    $skew-degrees: 10deg;

    display: flex;
    width: $item-width;
    height: $item-height;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    border-radius: $corner-custom;
    background-image: linear-gradient(180deg, $blue-3, $blue-2);
    text-align: center;
    position: relative;

    &:nth-child(2) {
      $is-middle: 1;

      @include skew-container($width: $item-max-width, $height: $item-height, $position: relative, $display: flex);
      @include skew-display($skewX: 10deg, $middle: $is-middle);
    }

    &:nth-child(1)::after {
      @include skew-container($width: $skew-width, $display: block);
      @include skew-display($right: $item-skew-distance, $skewX: -$skew-degrees, $zIndex: $z-index-under-layer);
      // unknown but this fixes an overwriting error
      transform: skewX(-$skew-degrees);
    }

    &:nth-child(3)::before {
      @include skew-container($width: $skew-width);
      @include skew-display($left: $item-skew-distance, $skewX: $skew-degrees, $zIndex: $z-index-under-layer);
    }

    .image-24 {
      margin-bottom: $v-12;
    }

    &__title {
      font-weight: bold;
    }

    &__subtitle {
      margin-top: $v-6;
      font-size: $font-mobile-size;
      line-height: $line-height-mobile;
      font-style: italic;
      max-width: 220px;
    }
  }
}
.about-section {
  $about-max-width: 940px;
  $v-60: $v-30 * 2;
  $corner-custom: 20px;

  max-width: $about-max-width;
  margin: auto;
  margin-top: $v-60;
  padding: $v-60 $h-120;
  border-radius: $corner-custom;
  background-image: linear-gradient(180deg, $gray-6, rgba($white, 0));
  font-family: $font-content;
  font-size: $font-regular-size;
  line-height: $line-height-regular;

  .about__title {
    margin-bottom: $v-12;
    font-family: 'Hind Vadodara', sans-serif;
    font-size: $h4-size;
    line-height: $h4-line-height;
  }
}

@media (max-width: 991px) {
  .values-container {
    align-items: center;
    justify-content: space-between;
    background-position: 0px 50%;
    background-size: contain;
    background-repeat: no-repeat;

    .values-item {
      position: relative;
      z-index: $z-index-2;
      margin: 0 $h-10;
      padding: $v-20;
    }
  }

  .career-section {
    display: flex;
    margin-bottom: 0px;
    float: none;
    justify-content: center;

    .tile {
      position: static;
      float: none;
      clear: none;
    }
  }
}

@include tablet {
  .values-container {
    flex-direction: row;
    align-items: stretch;
    background-image: none;
    background-size: auto;
    background-repeat: repeat;

    .values-item {
      width: 100%;
      height: auto;
      margin-bottom: $v-20;
      flex: 0 auto;
    }
  }

  .about-section {
    margin-top: $v-30;
    padding: $v-30 $h-40;
    font-size: $font-mobile-size;
    line-height: $line-height-mobile;

    &__title {
      font-size: $h4-size;
      line-height: $h4-line-height;
    }
  }

  .career-section {
    $section-width-max: 520px;
    $v-60: $v-30 * 2;

    .tile {
      display: flex;
      max-width: $section-width-max;
      margin-top: $v-60;
      margin-right: auto;
      margin-left: auto;
      padding: $h-40;
      float: none;
    }
  }
}

@include phone {
  .values-container {
    flex-direction: column;
    align-items: center;

    .values-item {
      max-width: 100%;
      padding-bottom: $v-36;
    }
  }

  .about-section {
    padding: $v-20;

    &__title {
      margin-top: 0px;
    }
  }

  .profile {
    margin-top: $v-24;
    $v-22: 22px;
    $v-26: 26px;
    $v-4: 4px;
    $line-height-22: 22px;

    .linkedin-link {
      height: $v-22;
      margin-top: $v-4;
      padding-left: $v-26;
      background-size: $v-20 $h-20;
      line-height: $line-height-22;
    }
  }

  .career-section {
    .tile {
      margin: $v-30 0;
      padding: $v-20;

      .button {
        max-width: 100%;
      }
    }
  }
}

.special-section {
  margin: 0 auto;
  display: flex;
  position: relative;
  @include tablet() {
    flex-direction: column;
  }
}

.profiles {
  $h-270: 270px;

  min-width: $h-270;
  @include tablet() {
    display: flex;
    justify-content: flex-start;
  }
  @include phone() {
    flex-direction: column;
  }
}
.profile {
  display: flex;
  margin-top: $h-40;
  align-items: flex-start;
  @include tablet() {
    margin-right: $h-40;
  }
  @include phone() {
    margin-right: unset;
  }
  .profile__info {
    margin-left: $h-20;

    .profile__title {
      color: #787e75;
      font-size: $font-mobile-size;
      line-height: $line-height-mobile;
      font-style: italic;
    }

    .linkedin-link {
      $v-26: 26px;
      $h-29: 29px;

      display: block;
      height: $v-26;
      margin-top: $v-12;
      padding-left: $h-29;
      background-image: url('../../assets/linkedin.svg');
      background-position: 0px 50%;
      background-repeat: no-repeat;
      color: #187fb8;
      line-height: $v-26;
    }
  }
}
.tile {
  $h-620: 620px;
  $h-30: $h-20 + $h-10;
  $v-320: 320px;
  $v-40: $v-20 * 2;

  z-index: $z-index-2;
  position: relative;
  min-width: $h-620;
  margin-left: $h-30;
  display: flex;
  margin-top: $v-40;
  padding: $v-48 $h-60 $h-60;
  flex-direction: column;
  align-items: center;
  border: 1px solid $gray-4;
  border-radius: $corner-soft;
  background-color: $white;
  box-shadow: 0 5px 15px 0 rgba(black, 0.1);
  text-align: left;
  @include tablet() {
    width: 100%;
    margin-left: unset;
  }
  @include phone() {
    min-width: unset;
    padding: $h-30;
  }
  &__description {
    display: block;
    max-width: $v-320;
    margin: $v-16 auto $h-30;
    font-size: $h5-size;
    text-align: center;
    @include phone() {
      min-width: unset;
      font-size: $font-mobile-size;
    }
  }

  &__logo {
    border-radius: $corner-soft;
  }
}
