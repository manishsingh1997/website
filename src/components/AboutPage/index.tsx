import React from 'react';

import {Button} from '@ergeon/core-components';
import imgHeart from '@ergeon/core-components/src/assets/icon-heart-favicon-blue.svg';
import imgJenny from '../../assets/about-page/jenny@2x.jpg';
import imgOdysseas from '../../assets/about-page/odysseas@2x.jpg';
import imgLogoMark from '../../assets/about-page/logo_mark.png';

import './index.scss';

class AboutPage extends React.Component {
  render() {
    return (
      <div className="wrapper-1180 spacing before__is-88">
        <div className="spacing before__is-48 after__is-48">
          <h1 className="center spacing after__is-12">Our mission</h1>
          <p className="subheader h2 center">Empower the world to build.</p>
        </div>
        <div className="values-container">
          <div className="values-item">
            <img alt="" className="image-24" src={imgHeart} />
            <h5>We invest in people</h5>
            <p className="values-item__subtitle">
              Making long term commitments to customers and team members.
              <br />
            </p>
          </div>
          <div className="values-item">
            <img alt="" className="image-24" src={imgHeart} />
            <h5 className="values-item__title">We are lean</h5>
            <p className="values-item__subtitle">Maximizing value for customers by minimizing waste.</p>
          </div>
          <div className="values-item">
            <img alt="" className="image-24" src={imgHeart} />
            <h5 className="values-item__title">We are kind</h5>
            <p className="values-item__subtitle">Setting new standards for fairness, empathy and respect.</p>
          </div>
        </div>
        <div className="about-section">
          <h3 className="spacing after__is-24">About Ergeon</h3>
          <p>
            We are disrupting the multi-trillion-dollar construction industry with a global skilled workforce and
            technology. We empower our customers, contractors, and staff to become their own superheroes.
            <br />
            <br />
            We believe that projects should be done on time, on budget, and with reliable quality. Our customers are
            provided with the best support from a highly qualified global workforce and they have access to
            user-friendly, proprietary technology allowing them to visualize, plan and build their projects. This opens
            the door to fast, responsive, and friendly service, transparent pricing, and quality craftsmanship. As a
            result, any property owner can make their home better without any stress; and enjoy it more with their
            family and friends.
            <br />
            <br />
            Starting with outdoor construction, we install the highest quality fence projects and we are on track toward
            scaling to larger complex building projects. Over 8,000 happy customers are now enjoying the fences weâ€™ve
            built for them. We were named Best of HomeAdvisor 2020, with over 2,000 public reviews at an average rating
            of 4.6. We have expanded across multiple states in the past four years to cover over 12% of U.S. households
            in our service areas! Our company was founded and is run today by two CS Ph.D. serial entrepreneurs.
            <br />
            <br />
          </p>
          <div className="special-section">
            <div className="profiles">
              <div className="profile">
                <img alt="" className="profile__image" src={imgJenny} width="96" />
                <div className="profile__info">
                  <h5>Jenny He</h5>
                  <div className="label">
                    <i>Founder and CEO</i>
                  </div>
                  <a
                    className="linkedin-link"
                    href="https://www.linkedin.com/in/jiayue-jenny-he-5a10762"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Linkedin
                  </a>
                </div>
              </div>
              <div className="profile">
                <img alt="" className="profile__image" src={imgOdysseas} width="96" />
                <div className="profile__info">
                  <h5>Odysseas Tsatalos</h5>
                  <div className="label">
                    <i>Founder and CTO</i>
                  </div>
                  <a
                    className="linkedin-link"
                    href="https://www.linkedin.com/in/tsatalos"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Linkedin
                  </a>
                </div>
              </div>
            </div>
            <div className="tile">
              <img alt="" className="tile__logo" src={imgLogoMark} width="67" />
              <p className="tile__description">
                If you crave an opportunity for impact and rapid career growth, join us!
              </p>
              <Button asAnchor className="min-width__is-120" flavor="secondary" href="/careers" size="large">
                Join us!
              </Button>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default AboutPage;
