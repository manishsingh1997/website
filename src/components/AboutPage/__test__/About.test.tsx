import React from 'react';

import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { MemoryRouter } from 'react-router-dom';

import AboutPage from '../index';

describe('About page', () => {
  test('It matches snapshot on render', () => {
    const { container} = render(
      <MemoryRouter>
        <AboutPage />
      </MemoryRouter>
    );
    
    expect(container).toMatchSnapshot();
  });
});
