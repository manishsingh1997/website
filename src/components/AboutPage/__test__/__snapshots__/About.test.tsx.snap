// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page It matches snapshot on render 1`] = `
<div>
  <div
    class="wrapper-1180"
  >
    <div
      class="spacing before__is-48 after__is-48"
    >
      <h1
        class="center spacing after__is-12"
      >
        Our mission
      </h1>
      <p
        class="subheader h2 center"
      >
        Empower the world to build.
      </p>
    </div>
    <div
      class="values-container"
    >
      <div
        class="values-item"
      >
        <img
          alt=""
          class="image-24"
          src="icon-heart-favicon-blue.svg"
        />
        <h5>
          We invest in people
        </h5>
        <p
          class="values-item__subtitle"
        >
          Making long term commitments to customers and team members.
          <br />
        </p>
      </div>
      <div
        class="values-item"
      >
        <img
          alt=""
          class="image-24"
          src="icon-heart-favicon-blue.svg"
        />
        <h5
          class="values-item__title"
        >
          We are lean
        </h5>
        <p
          class="values-item__subtitle"
        >
          Maximizing value for customers by minimizing waste.
        </p>
      </div>
      <div
        class="values-item"
      >
        <img
          alt=""
          class="image-24"
          src="icon-heart-favicon-blue.svg"
        />
        <h5
          class="values-item__title"
        >
          We are kind
        </h5>
        <p
          class="values-item__subtitle"
        >
          Setting new standards for fairness, empathy and respect.
        </p>
      </div>
    </div>
    <div
      class="about-section"
    >
      <h3
        class="spacing after__is-24"
      >
        About Ergeon
      </h3>
      <p>
        We are disrupting the multi-trillion-dollar construction industry with a global skilled workforce and technology. We empower our customers, contractors, and staff to become their own superheroes.
        <br />
        <br />
        We believe that projects should be done on time, on budget, and with reliable quality. Our customers are provided with the best support from a highly qualified global workforce and they have access to user-friendly, proprietary technology allowing them to visualize, plan and build their projects. This opens the door to fast, responsive, and friendly service, transparent pricing, and quality craftsmanship. As a result, any property owner can make their home better without any stress; and enjoy it more with their family and friends.
        <br />
        <br />
        Starting with outdoor construction, we install the highest quality fence projects and we are on track toward scaling to larger complex building projects. Over 8,000 happy customers are now enjoying the fences weâ€™ve built for them. We were named Best of HomeAdvisor 2020, with over 2,000 public reviews at an average rating of 4.6. We have expanded across multiple states in the past four years to cover over 12% of U.S. households in our service areas! Our company was founded and is run today by two CS Ph.D. serial entrepreneurs.
        <br />
        <br />
      </p>
      <div
        class="special-section"
      >
        <div
          class="profiles"
        >
          <div
            class="profile"
          >
            <img
              alt=""
              class="profile__image"
              src="jenny@2x.jpg"
              width="96"
            />
            <div
              class="profile__info"
            >
              <h5>
                Jenny He
              </h5>
              <div
                class="label"
              >
                <i>
                  Founder and CEO
                </i>
              </div>
              <a
                class="linkedin-link"
                href="https://www.linkedin.com/in/jiayue-jenny-he-5a10762"
                rel="noopener noreferrer"
                target="_blank"
              >
                Linkedin
              </a>
            </div>
          </div>
          <div
            class="profile"
          >
            <img
              alt=""
              class="profile__image"
              src="odysseas@2x.jpg"
              width="96"
            />
            <div
              class="profile__info"
            >
              <h5>
                Odysseas Tsatalos
              </h5>
              <div
                class="label"
              >
                <i>
                  Founder and CTO
                </i>
              </div>
              <a
                class="linkedin-link"
                href="https://www.linkedin.com/in/tsatalos"
                rel="noopener noreferrer"
                target="_blank"
              >
                Linkedin
              </a>
            </div>
          </div>
        </div>
        <div
          class="tile"
        >
          <img
            alt=""
            class="tile__logo"
            src="logo_mark.png"
            width="67"
          />
          <p
            class="tile__description"
          >
            If you crave an opportunity for impact and rapid career growth, join us!
          </p>
          <a
            class="button button--cta button--size__large taste__solid min-width__is-120"
            href="/careers"
          >
            Join us!
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
